fe1-web/src/features/lao/functions/test-connection.ts

```diff
@@ -0,0 +1,27 @@
+import { KeyPairStore } from 'core/keypair';
```

Comment:
The `test-connection.ts` filename doesn't seem to match the usual naming practice

Reply:
👍